app/assets/javascripts/swagger-ui/threescale.coffee.erb

```diff
@@ -94,11 +98,10 @@ class ThreeScaleHttp
     @linkElement
 
   sameOrigin: ->
-    <% if Rails.configuration.three_scale.active_docs_proxy_disabled %>
-      true
-    <% else %>
+    if @enableApiDocsProxy()
       @desiredOrigin() == @locationOrigin()
-    <% end %>
+    else
+      true
```

Comment:
!@enableApiDocsProxy() |@desiredOrigin() == @locationOrigin()

Reply:
You mean `!@enableApiDocsProxy() || @desiredOrigin() == @locationOrigin()` ?